src/orderbook.js

```diff
@@ -164,7 +201,15 @@ OrderBook.EVENTS = [
   'offer_changed', 'offer_funds_changed'
 ];
 
-OrderBook.DEFAULT_TRANSFER_RATE = 1000000000;
+OrderBook.DEFAULT_TRANSFER_RATE = Amount.from_json(1000000000);
+
+OrderBook.NOTIFY_THROTTLE_TIME = 4000;
```

Comment:
In this implementation leg book 2's modifications will be reflected after `AUTOBRIDGE_CALCULATE_THROTTLE_TIME` time will pass - specifically for that I've used
`{leading: true, trailing: true}` as `_.throttle` options - so `computeAutobridgedOffersWrapper` will be called at the end of period in this case.
And we cannot use debounce here, because we risk to never see changes if there will be high volume of transactions.